src/components/pages/photos/elements/RoverImage.js

```diff
@@ -1,14 +1,41 @@
 import React, { Component } from 'react';
 import CameraNavbar from './cameraNavbar';
-import nisa from '../../../../assets/images/nisa.JPG';
 import '../PhotoPage.css';
+import RoverImageButton from './RoverImageButton';
 
 export default class RoverImage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      index: 0,
+    };
+  }
+
+  incrementIndex() {
+    this.setState(state => (
+      {
+        index: Math.min(state.index + 1, this.props.photoUrlArray.length - 1),
+      }
+    ));
+  }
+
+  decrementIndex() {
+    this.setState(state => (
+      {
+        index: Math.max(0, state.index - 1),
+      }
+    ));
+  }
+
+
   render() {
     return (
       <div className="rover-image">
-        <img className="camera-capture" src={nisa} alt="Mars" />
+        <RoverImageButton className="leftButton" direction={'<'} onClick={() => this.decrementIndex()} />

```

Comment:
Another thing to think about, when we have an element with a lot of properties it can be useful (and increase readability) to split the properties out onto their own lines:
```
<RoverImageButton
    className="leftButton"
    direction={'<'}
    onClick={() => this.decrementIndex()}
/>
```
This also applies to the `<button>` element in `RoverImageButton`.

Reply:
I also don't think the `{}` are necessary for direction - we should be able to write `direction="<"`